docs/source/bare-metal.rst

```diff
@@ -134,6 +135,82 @@ If you check the felix logfile after this step, the logs should transition from
 periodic notifications that felix is in state "wait-for-ready" to a stream
 of initialisation messages.
 
+.. _`Creating basic connectivity and Calico policy`:
+
+Creating basic connectivity and Calico policy
+---------------------------------------------
+
+When a host endpoint is added, if there is no security policy for that
+endpoint, Calico will default to denying traffic to/from that endpoint, except
+for traffic that is allowed by the `failsafe rules`_.
+
+While the `failsafe rules`_ provide some protection against removing all
+connectivity to a host, they are also quite broad and do not include all the
+connectivity that a typical host requires.  Therefore, we recommend
```

Comment:
That last clause is a little tricky to grasp, because it seems to say 'they are too broad and too narrow'.  Perhaps: "they are overly broad in the SSH access that they allow, yet may not include all the connectivity that a typical host requires" ?